File: app/src/main/java/example/com/crackle/model/CreditResults.java
package example.com.crackle.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import example.com.crackle.model.Cast;
import example.com.crackle.model.Crew;

public class CreditResults {

    @SerializedName("id")
    private int id;
    @SerializedName("cast")
    private List<Cast> castList;
    @SerializedName("crew")
    private List<Crew> crewList;

    public CreditResults(int id, List<Cast> castList, List<Crew> crewList) {
        this.id = id;
        this.castList = castList;
        this.crewList = crewList;
    }

    public int getId() {
        return id;
    }

    public List<Cast> getCastList() {
        return castList;
    }

    public List<Crew> getCrewList() {
        return crewList;
    }
}

File: app/src/main/java/example/com/crackle/model/CreditResults.kt
package example.com.crackle.model

import com.google.gson.annotations.SerializedName

data class CreditResults(@field:SerializedName("id")
                    val id: Int, @field:SerializedName("cast")
                    val castList: List<Cast>, @field:SerializedName("crew")
                    val crewList: List<Crew>)
